.github/PULL_REQUEST_TEMPLATE/cypress_repro.md

```diff
@@ -1,31 +1,25 @@
 ### Status
-
 PENDING CI / PENDING REVIEW / READY _(choose one and update accordingly)_
 
 ### What does this PR accomplish?
-
 - Reproduces #XXXXX
 
 ### How to test this manually?
-
 - `yarn test-cypress-open`
-- `relative/path/to/the/file` _(optionally, include the line number on which the test starts)_
+- `relative/path/to/the/file`
```

Comment:
There are some duplicate spec names living under different folders. That's why it's helpful to include the whole path.

Hint: 
- You can paste that same path in Cypress UI's search field.
![ ](https://p58.f1.n0.cdn.getcloudapp.com/items/4gu1Ddbv/2aae74a7-c86b-44ee-9cbf-2194ea19dee0.gif?v=2f598cbbb667f07bdb29794eb0525549)